plugins/logs/plugin.go

```diff
@@ -385,6 +415,37 @@ func (p *Plugin) bufferChunk(buffer *logBuffer, bs []byte) {
 	}
 }
 
+func (p *Plugin) maskEvent(ctx context.Context, event *EventV1) error {
+
+	query := ast.NewBody(ast.NewExpr(ast.NewTerm(p.config.maskDecisionRef)))
+
+	r := rego.New(
```

Comment:
This is running policy for each event, right? Is that going to be an issue (CPU)?